index.html

```diff
@@ -116,17 +115,12 @@ <h2 class="tonic">Tonic</h2>
 
    
    <div class="section4">
-    <row>
-      <col>
-    <img src="images/d.png" class="imagesec4"></div>
-    </col>
-
     
-    <col>
+    <img src="images/d.png" alt="ss" class="imagesec4"></div>
+    
     <h2 class="tonic">Multi-Post
       Stories</h2>
-                       
-
+                       <ul>
                       <li class="sec3rlink2">
         
         <a href="/someurl" class="sec3rlink3">CANOPY</a>

```

Comment:
- [ ] [REQUIRED] Kindly, please, make sure that the cards sections is similar to the Figma design. Try to use the same fonts and same colors and also same border styles.

Your Design | Figma Design
------------|-------------
![image](https://user-images.githubusercontent.com/60979926/178114362-62581e01-ce15-4aa1-921f-df529f39f253.png) | ![image](https://user-images.githubusercontent.com/60979926/178114475-d4ded510-046d-4797-bc45-3ed911dfd4ce.png)

Reply:
@azizunnabi I can see you haven't made the changes requested by the previous reviewer, please implement the changes requested.